pkg/email/send_email.go

```diff
@@ -11,30 +14,35 @@ type Emailer struct {
 	Limiter *rate.Limiter
 }
 
+var (
+	ErrRateLimitExceeded = errors.New("rate limit exceeded")
+)
```

Comment:
just style for single var
```suggestion
var ErrRateLimitExceeded = errors.New("rate limit exceeded")
```

Reply:
Fixed